sdk/storage/storage-blob/samples/javascript/advanced.js

```diff
@@ -36,9 +36,9 @@ class ConsoleHttpPipelineLogger {
 
 async function main() {
   // Fill in following settings before running this sample
-  const account = "";
-  const accountSas = "";
-  const localFilePath = "";
+  const account = process.env.ACCOUNT_NAME || "";
+  const accountSas = process.env.ACCOUNT_SAS || "";
+  const localFilePath = "../README.md";
```

Comment:
nit: could upload the source code file itself?

Reply:
Which source code file?

Reply:
"./advanced.js"

Reply:
Test file names may change, readme will be present forever.

Reply:
fair, I was thinking there could be folder changes. This is good to merge.